README.md

```diff
@@ -12,12 +12,15 @@ Apache-2.0
 
 
 ## Set up
-    ```
-    rustup default nightly-2020-10-06
-    ```
 
-    ```
-    cargo build
-    ```
+```
+rustup update
+rustup update nightly
+rustup target add wasm32-unknown-unknown --toolchain nightly-2020-10-06
```

Comment:
This should be outdated as we need to use the latest nightly since substrate 2.1.